linen_examples/seq2seq/train.py

```diff
@@ -352,24 +302,35 @@ def compute_metrics(logits, labels):
   return metrics
 
 
+IN_SHAPES = [{'query': '(n, _)', 'answer': '(m, _)'}]
+OUT_ELEM = f'(m +- 1, {CTABLE.vocab_size})'
```

Comment:
curious: what does `m +- 1` syntax mean here?

Reply:
This is the somewhat funny notation to denote "m-1". The decoder outputs "m-1" tokens since the input also contains the start token "=".

Reply:
ok cool that makes sense.

Reply:
Minor nit : Could we make it `m + -1` then? Then it's clearer that we add `-1` as opposed to inventing a new operator `+-` with mysterious semantics :)

Reply:
Sure, good idea! Done.